### PR TITLE
main/krb5: improve

### PR DESCRIPTION
**Pull request by tcely**

### CODE REVIEW

File: main/krb5/APKBUILD [hunk 1]
```diff
@@ -19,14 +13,23 @@ makedepends="$depends_dev libverto-dev openldap-dev openssl-dev
...
+_ver_maj_min() {
+	local v
+	for v; do
+		while true; do
+			case "${v}" in (*.*.*) v="${v%.*}" ;; (*.*) break ;; esac
+		done
+		printf -- '%s\n' "${v}"
+	done
+}
+_v="$(_ver_maj_min "${pkgver}")"
```
Comment (ncopa) [inline main/krb5/APKBUILD]: I don't want to have forks in global scope.

Comment (tcely) [reply main/krb5/APKBUILD]: I reworked this to not use a sub-shell.

**Comment by ncopa** [comment]
I think this is needlessly complicated. I will use some of the hunks and clean up a bit.

**Comment by ncopa** [comment]
superseded by 578a855aebad71b98dfa551a5c7a7ce942e8d254

**Pull request closed without merging**